sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -876,7 +869,7 @@ def test_top_level(self):
                 'merge_ordered(df1, df2, fill_method="ffill", left_by="group")'
             ],
             # Expected error.
-            'pivot': ["df.pivot(index='foo', columns='bar', values='baz')"],
+            'pivot': ["*"],
```

Comment:
I don't think we want to skip all the pivot tests, can we revert this line and instead add `'pivot': ["*"]` to wont_implement_ok (as suggested above).

Reply:
Done!